frontend/src/components/Pages/MissionHistoryPage/MissionHistoryView.tsx

```diff
@@ -33,31 +62,56 @@ export function MissionHistoryView({ refreshInterval }: RefreshProps) {
     const [isLoading, setIsLoading] = useState<boolean>(true)
     const [isResettingPage, setIsResettingPage] = useState<boolean>(false)
 
-    const {
-        page,
-        switchPage,
-        missionName,
-        statuses,
-        robotName,
-        tagId,
-        inspectionTypes,
-        minStartTime,
-        maxStartTime,
-        minEndTime,
-        maxEndTime,
-    } = useMissionFilterContext()
+    const { page, switchPage, filterState, filterIsSet, filterFunctions, filterError, clearFilterError } =
+        useMissionFilterContext()
+
+    const FilterErrorDialog = () => {
+        return (
+            <>
+                <Dialog open={filterError != ''} isDismissable onClose={() => clearFilterError()}>
+                    <Dialog.Header>
+                        <Dialog.Title>Dismissable dialog</Dialog.Title>
+                    </Dialog.Header>
+                    <Dialog.CustomContent>
+                        <Typography variant="body_short">Closes dialog on click outside and escape key.</Typography>
+                    </Dialog.CustomContent>
+                    <Dialog.Actions>
+                        <Button onClick={() => clearFilterError()}>OK</Button>
+                    </Dialog.Actions>
+                </Dialog>
+            </>
+        )
+    }
+
+    const toDisplayValue = (filterName: string, value: string | number | MissionStatus[] | InspectionType[]) => {
+        if (typeof value === 'string') {
+            return value
+        } else if (typeof value === 'number') {
+            // We currently assume these are dates. We may want
+            // to explicitly use the Date type in the filter context instead
+            return filterFunctions.dateTimeIntToPrettyString(value)
+        } else if (Array.isArray(value)) {
+            let valueArray = value as any[]
+            if (valueArray.length === 0) {
+                console.error('Unexpected empty array detected')
+            }
+            return valueArray.map((val) => (
+                <Chip
+                    key={filterName + val}
+                    variant="error"
+                    onDelete={() => filterFunctions.removeFilterElement(filterName, val)}
```

Comment:
You can try to add here style={{ border: tokens.colors.interactive.primary_resting.rgba }}

Reply:
![image](https://github.com/equinor/flotilla/assets/108514046/d4fa631a-0ac0-4dd8-b65b-425c2825a80d)

something like this